guides/common/modules/proc_salt-configuring-salt-master.adoc

```diff
@@ -0,0 +1,60 @@
+[id="salt_guide_configuring_salt_master_{context}"]
+= Configuring Salt Master
+
+Configure your Salt Master to configure managed hosts using Salt.
+
+.Procedure
+. Connect to your Salt Master using SSH:
++
+[options="nowrap" subs="attributes"]
+----
+# ssh root@salt-master.example.com
+----
+. Set {Project} as an external node classifier in `/etc/salt/master` for Salt:
++
+[options="nowrap" subs="attributes"]
+----
+master_tops:
+  ext_nodes: /usr/bin/foreman-node
+----
+. Enable Salt Pillar data for use with {Project}:
++
+[options="nowrap" subs="attributes"]
+----
+ext_pillar:
+  - puppet: /usr/bin/foreman-node
+----
+. Add a Salt Environment called `base` which is associated with the `/srv/salt` directory:
```

Comment:
```suggestion
. Add a Salt Environment called `base` which is associated with the `/srv/salt` directory:
```

I corrected the E in another comment. Is this a term in Salt ?

Reply:
Yes, it's Salt terminology.